akka-docs/src/main/paradox/cluster-dc.md

```diff
@@ -158,8 +158,8 @@ Scala
 Java
 :  @@snip [ClusterSingletonManagerTest.java]($akka$/akka-cluster-tools/src/test/java/akka/cluster/singleton/ClusterSingletonManagerTest.java) { #create-singleton-proxy-dc }
 
-If using the own data center as the `withDataCenter` parameter that would be a proxy for the singleton in the own data center, which
-is also the default if `withDataCenter` is not given.
+If using the own data center as the @scala[@scaladoc[`withDataCenter`](akka.cluster.singleton.ClusterSingletonProxySettings)]@java[@javadoc[`withDataCenter`](akka.cluster.singleton.ClusterSingletonProxySettings)] parameter that would be a proxy for the singleton in the own data center, which
+is also the default if @scala[@scaladoc[`withDataCenter`](akka.cluster.singleton.ClusterSingletonProxySettings)]@java[@javadoc[`withDataCenter`](akka.cluster.singleton.ClusterSingletonProxySettings)] is not given.
```

Comment:
Similar to [this comment](https://github.com/akka/akka/pull/24525/files#r167432330), scaladoc to a method includes return type, which is messy. Similar reason for javadoc.